workflows/cloud-client/main.py

```diff
@@ -48,25 +51,21 @@ def execute_workflow(
     # Wait for execution to finish, then print results.
     execution_finished = False
     backoff_delay = 1  # Start wait with delay of 1 second
-    print('Poll every second for result...')
-    while (not execution_finished):
-        execution = execution_client.get_execution(
-            request={"name": response.name})
+    print("Poll every second for result...")
```

Comment:
This is wrong. It's Polling, but not every second. could just say Poll for result instead.